src/components/Matching/Customizer.vue

```diff
@@ -119,7 +117,7 @@ function handleEditChoice(nodeId) {
   if (!res) return
   const newChoice = isUUID(res)
     ? { type: 'image', imageId: res, nodeId } 
-    : { type: 'text', textContent: res, nodeId }
+    : { type: 'audio', audioId: res, nodeId}
```

Comment:
Looks like we lost the ability to add text content. We might want to allow that in a generic customizer. Let's go ahead and get this version ready to merge, and we can add back in text content later